### DISCUSSION TITLE
Add Logo and Text in stream video

### DISCUSSION THREAD

**Discussion by msman0** [owncast/owncast#2950] [Q&A]
Hi, how can I add a logo in the video stream and a text. The text should be dynamic read from a file at the server.
Thank you!

**Comment by gabek** [comment]
This wouldn't be something that Owncast would do, you'd do this from the broadcasting side. Perhaps look into: https://github.com/Erkaman/ffmpeg-add-text-to-video-tutorial

**Comment by msman0** [comment]
Thank you very much for the quick reply. Perhaps you can add the feature sometime, it would be great.

**Comment by rmens** [comment]
OBS Studio is great for this!

**Reply by msman0** [reply]
Yes but I have only a Webcam with the stream and no PC/Mac etc. between the stream to add it via OBS. It should work without interaction needed.
